rust/connlib/shared/src/lib.rs

```diff
@@ -68,16 +68,16 @@ pub fn get_user_agent(os_version_override: Option<String>, app_version: &str) ->
     let os_version = os_version_override.unwrap_or(info.version().to_string());
     let additional_info = additional_info();
     let lib_name = LIB_NAME;
-    format!("{os_type}/{os_version}{additional_info}{lib_name}/{app_version}")
+    format!("{os_type}/{os_version} {lib_name}/{app_version}{additional_info}")
```

Comment:
Fixes the order and format of the user_agent on all OSes.

Reply:
Does the portal decode this? Is this a breaking change now that we select the gateway based on the version?

Reply:
It does, but it checks for `connlib/{version}` so this should be OK.